contrib/kube-prometheus/docs/How-to-monitor-an-external-secure-etcd-cluster.md

```diff
@@ -0,0 +1,164 @@
+# How to monitor a secure external etcd service with Prometheus Operator
+This guide will help you monitor an external etcd cluster. When the etcd is not hosted inside Kubernetes.
+This is often the case with the Kubernetes setup. This has been tested with kube-aws but same principals will apply to other tools.
```

Comment:
This is often the case with Kubernetes setups. This approach has been tested with kube-aws but the same principals apply to other tools.